src/views/results/index.ts

```diff
@@ -142,6 +142,40 @@ export interface ParsedStatementInfo extends StatementInfo {
 
 export function initialise(context: vscode.ExtensionContext) {
   let resultSetProvider = new ResultSetPanelProvider();
+  let selfCodeErrorProvider = new SelfCodePanelProvider();
+
+  context.subscriptions.push(
+    vscode.window.registerWebviewViewProvider(`vscode-db2i.selfCodeErrorPanel`, selfCodeErrorProvider, {
+      webviewOptions : {retainContextWhenHidden: true}
+    }),
+    vscode.commands.registerCommand(`vscode-db2i.selfCodeErrorPanel.fetch`, async (job: SQLJob, showErrors: boolean) => {
+      await selfCodeErrorProvider.ensureActivation();
+      const content = `SELECT * FROM QSYS2.SQL_ERROR_LOG WHERE JOB_NAME = '${job.id}'`;
+      let prevDataLength = job.selfCodeCounter;
+      let updateView: boolean = false;
+      let data = [];
+      for (let count = 0; count < 5; count++) {
+        const curData = await JobManager.runSQL(content, undefined);
```

Comment:
Instead of using the `JobManager` here, you should just use the attached `job` parameter (`job.query`)